model/src/main/kotlin/OrtResult.kt

```diff
@@ -94,6 +94,10 @@ data class OrtResult(
 
     private data class ProjectEntry(val project: Project, val isExcluded: Boolean)
 
+    /**
```

Comment:
commit: As this area had tricky performance issues in the past would you mind testing this change against a large project?

Reply:
Yes, will do.

Reply:
Test is finished, all commands took the same time as before this change, except for the reporter which was slightly faster after this change.

Reply:
Thanks!